config/Migrations/20213028210100_add_whmcs_fields.php

```diff
@@ -0,0 +1,26 @@
+<?php
+use Migrations\AbstractMigration;
+
+class AddWhmcsFields extends AbstractMigration
+{
+
+    public $autoId = false;
+
+    public function change()
+    {
+        $table = $this->table('users');
+        $table
+            ->addColumn('whmcs_real_user_id', 'integer', [
+                'default' => null,
+                'limit' => 10,
+                'null' => true,
+                'signed' => false,
+            ])
+            ->addColumn('whmcs_user_id_password', 'string', [
```

Comment:
This column name is somewhat confusing, what is it's purpose? I suggest either renaming it or at least adding comments explaining it's use.